README.md

```diff
@@ -8,6 +8,12 @@
 [![Gitter](https://badges.gitter.im/covid19-review/community.svg)](https://gitter.im/covid19-review/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 <!-- usage note: delete CI badges above for services not used by your manuscript -->
 
+## Code of Conduct
+
+Participation in this project requires that you adhere to the [code of conduct](CODE_OF_CONDUCT.md).
+We take this very seriously.
```

Comment:
```suggestion
Participating in the project in any way (issues, pull requests, gitter,  or other media) indicates that you agree that you will follow this code of conduct.
We take this very seriously.
```

Reply:
Do you want to keep the link to the code of conduct here?  It may not be clear what "this code" is otherwise.

Reply:
Agree - revised 👍